addons/infinite-scroll/userscript.css

```diff
@@ -15,7 +15,8 @@ body {
   top: initial;
   bottom: 0;
   width: 100%;
-  height: 46px;
+  border-top: 1px solid #b3b3b3;
+  height: 10px;
```

Comment:
10px is too small to easily access. scratch-www uses a height of 48px for most interactive elements, so please make it at least 32px.

For the border, I'd use `#d9d9d9` and add `#footer` to [this selector](https://github.com/ScratchAddons/ScratchAddons/blob/master/addons/dark-www/experimental_scratchwww.css#L110-L138).

Reply:
I know 10px is small, but it's still fixed at the bottom of the page? All you need to access it is move the cursor to the bottom of the viewport. Doesn't seem fair to compare it against interactive elements that do not have that behavior?
Also, are you OK adding a border to the footer on dark-www, even tho vanilla Scratch does not have a border there? It's only added by infinite scrolling.

Reply:
> I know 10px is small, but it's still fixed at the bottom of the page? All you need to access it is move the cursor to the bottom of the viewport. Doesn't seem fair to compare it against interactive elements that do not have that behavior?

I don't think being at the bottom of the page makes it easier to access with the cursor (except in full screen). It's close to the bottom of the screen but not at the bottom, which isn't very practical.

> Also, are you OK adding a border to the footer on dark-www, even tho vanilla Scratch does not have a border there? It's only added by infinite scrolling.

`border-color` doesn't add a border if it isn't already there, so adding `#footer` to the selector I linked in the previous comment should work.

Reply:
> except in full screen

What do you mean by that? On windows OS at least, if you move your cursor to the taskbar, it also shows the footer. You don't have to put the cursor within those 10 CSS pixels, you can move the cursor further down with no issue

> border-color doesn't add a border if it isn't already there, so adding #footer to the selector I linked in the previous comment should work.

Clever

Reply:
> > except in full screen
> 
> What do you mean by that? On windows OS at least, if you move your cursor to the taskbar, it also shows the footer. You don't have to put the cursor within those 10 CSS pixels, you can move the cursor further down with no issue

Doesn't work for me. Even if it did, it would only be possible if the window was maximized.

Reply:
Good point